core/indigo-core/molecule/ket_commons.h

```diff
@@ -98,5 +99,106 @@ namespace indigo
         std::list<KETTextLine> _block;
         Vec3f _pos;
     };
+
+    class KETReactionArrow : public MetaObject
+    {
+    public:
+        static const std::uint32_t cid = "KET reaction arrow"_hash;
```

Comment:
Maybe express that this is a constant by naming it ALL_CAPS: `CID` ?

Reply:
fixed